tests/test_enterprise/management/test_email_drip_for_missing_dsc_records.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for the djagno management command `email_drip_for_missing_dsc_records`.
+"""
+import random
+from datetime import timedelta
+
+import mock
+from pytest import mark
+
+from django.core.management import call_command
+from django.test import TestCase
+from django.utils import timezone
+
+from consent.models import ProxyDataSharingConsent
+from enterprise.management.commands.email_drip_for_missing_dsc_records import PAST_NUM_DAYS
+from test_utils.factories import (
+    EnterpriseCourseEnrollmentFactory,
+    EnterpriseCustomerFactory,
+    EnterpriseCustomerUserFactory,
+    UserFactory,
+)
+
+
+@mark.django_db
+class CreateEnterpriseCourseEnrollmentCommandTests(TestCase):
+    """
+    Test command `email_drip_for_missing_dsc_records`.
+    """
+    command = 'email_drip_for_missing_dsc_records'
+
+    def setUp(self):
+        super().setUp()
+
+        self.course_ids = [
+            'course-v1:edX+DemoX+Demo_Course',
+            'course-v1:edX+Python+1T2019',
+            'course-v1:edX+React+2T2019',
+        ]
+        self.enterprise_customer = EnterpriseCustomerFactory(
+            name='Starfleet Academy',
+            enable_data_sharing_consent=True,
+            enforce_data_sharing_consent='at_enrollment',
+        )
+
+        learners = []
+        for __ in range(5):
```

Comment:
can you update the test case such that, out of these 5, we have some learners who have consented to data sharing?

Reply:
Done